### PR TITLE
Task 19594 19605

### PR DESCRIPTION
**Pull request by BrandonHernandez-Ksquare**
models client and sale done

### CODE REVIEW

**Review by ClaudiaDiaz-Ksquare — COMMENTED** [review]
Maybe the references could be creation optional, and did you add the models to the models array (for initialization) inside the index file?

**Review by GenaroArredondo-Ksquare — APPROVED** [review]

**Review by ClaudiaDiaz-Ksquare — APPROVED** [review]

**Review by ClaudiaDiaz-Ksquare — COMMENTED** [review]
Just add the models inside the models array in the index file, for initizalization

**Review by ClaudiaDiaz-Ksquare — DISMISSED** [review]

File: src/models/Customer.model.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { Model, InferAttributes, InferCreationAttributes, CreationOptional, DataTypes, Sequelize } from "sequelize";
+
+
+export class Client extends Model<InferAttributes<Client>, InferCreationAttributes<Client>> {
+
+    declare id: number;
+    declare full_name: string;
+    declare phone_number: number;
+    declare user_id: number;
```
Comment (GabrielOsorno-Ksquare) [inline src/models/Customer.model.ts:9]: I think id should be CreationOptional and user_id a string. status could be CreationOptional too??

File: src/models/Customer.model.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            type: DataTypes.NUMBER,
+            allowNull: false,
+        },
+        user_id: {
+            type: DataTypes.NUMBER,
+            allowNull: false,
+        },
+        status: {
+            type: DataTypes.BOOLEAN,
+        }
```
Comment (GabrielOsorno-Ksquare) [inline src/models/Customer.model.ts:36]: Same, user_id to STRING, and status may have a dafaultValue. full_name could have allowNull: false

File: src/models/Sale.model.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    declare order_id: number;
+    declare total_price: number;
+    declare total_items: number;
+    declare date: Date;
+    declare payment_id: number; 
+    declare address: string;
+    declare city_id: number;
+    declare refrences: CreationOptional<boolean>;
+    declare is_deleted: CreationOptional<boolean>;
+
```
Comment (GabrielOsorno-Ksquare) [inline src/models/Sale.model.ts:16]: id should have CreationOptional

File: src/models/Sale.model.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            type: DataTypes.INTEGER
+        },
+        refrences: {
+            type: DataTypes.STRING
+        },
+        is_deleted: {
+            type: DataTypes.BOOLEAN,
+            defaultValue: false,
+        }
+
```
Comment (GabrielOsorno-Ksquare) [inline src/models/Sale.model.ts:56]: references should be allowNull: true I think

**Review by ClaudiaDiaz-Ksquare — APPROVED** [review]

**Review by GabrielOsorno-Ksquare — CHANGES_REQUESTED** [review]

File: src/models/Customer.model.ts (hunk 1 above)
Comment (BrandonHernandez-Ksquare) [reply src/models/Customer.model.ts:9]: why should user_id be a string?

**Review by GabrielOsorno-Ksquare — APPROVED** [review]

**Review by GenaroArredondo-Ksquare — APPROVED** [review]

**Review by ClaudiaDiaz-Ksquare — APPROVED** [review]

**Review by RodrigoLeon-Ksquare — APPROVED** [review]

**Pull request merged**